include/atari.h

```diff
@@ -159,6 +159,103 @@
 #define JOY_FIRE_MASK   JOY_BTN_1_MASK
 #define JOY_FIRE(v)     ((v) & JOY_FIRE_MASK)
 
+/* Keyboard values returned by kbcode / CH */
+#define KEY_NONE        (unsigned char) 0xFF
```

Comment:
I'm not sure if it's necessary because of the high precedence of the cast operator but my gut feeling asks for brackets around the term.

However, if "everybody else" says this is just stupid and only shows I haven't understood C then I'm perfectly willing to merge as-is.

Reply:
I'd prefer too to have brackets around the whole expression.
But, may I ask, why is the cast operator there? The other "constant defines" in atari.h don't have it.

Reply:
So that when they are used in void variables/arrays (cc65 extension) only eat up one byte instead of two. (Like the definitions in "_antic.h").

Reply:
Can you give an example where this makes a difference? (Don't want to talk about the _antic.h definiitions right now, I was never in favor of them...)

Reply:
`void keysToUpHandler = {KEY_W,KEY_UP,KEY_NONE,upHandler};` ?

About "_antic.h": Did you ever used the definitions?

Reply:
> About "_antic.h": Did you ever used the definitions?
No.
I guess it's just a matter of personal preference. To me, with the old definitions, something like `DL_CHR40x8x1 | DL_DLI` is immediately obvious, but when I see `DL_DLI(DL_CHR40x8x1)` I need to run another thinking thread along the lines "what is `DL_DLI()` exactly doing again?"